missions/Models/AudioPlayback.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AudioPlayback.swift
+//  missions
+//
+//  Created by Elina Lua Ming on 12/17/19.
+//  Copyright © 2019 ukaton. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+@available(iOS 13, *)
+final class AudioPlayback: NSObject, AVAudioPlayerDelegate {
```

Comment:
Source: if you got this code from stackoverflow or a blog, be sure to add a comment above the class declaration that has a link to the source.